base.gradle

```diff
@@ -206,6 +206,7 @@ dependencies {
     implementation androidLibs.composeUiUtil
     implementation androidLibs.wearPlayServices
 
+    implementation libs.turbine
```

Comment:
I think we can change this to `testImplementation` (and move it down with the other test dependencies). That way this doesn't get bundled into the app's apk.